### PR TITLE
Add test sync status persists after task delete

### PR DESCRIPTION
**Pull request by swadeley**
Hi

Adding test for bug[1].

[1]
[Bug 1924625](https://bugzilla.redhat.com/show_bug.cgi?id=1924625) - Sync status showing never synced even though the repositories has been synced successfully

### CODE REVIEW

File: tests/foreman/ui/test_repository.py [hunk 1]
```diff
@@ -1039,6 +1042,64 @@ def test_positive_remove_credentials(session, function_product, function_org, fu
...
+
+    :customerscenario: true
+
+    :Steps:
+        1. Sync a custom Repo.
+        2. Navigate to Content > Sync Status. Assert status is Synced.
+        3. Use foreman-rake console to delete the Sync task.
+        4. Navigate to Content > Sync Status. Assert status is still Synced.
+
+    :expectedresults: Sync Repositories should be shown as Synced on Sync Status page even
```
Comment (mshriver) [inline tests/foreman/ui/test_repository.py:1062]: expectedresults needs to be a list if its going to span multiple lines, this is failing testimony validation.

Comment (swadeley) [reply tests/foreman/ui/test_repository.py:1062]: oh, ok

File: tests/foreman/ui/test_repository.py [hunk 2]
```diff
@@ -1039,6 +1042,64 @@ def test_positive_remove_credentials(session, function_product, function_org, fu
...
+    after tasks are deleted
+    """
+    # make a note of time for later API wait_for_tasks, and include 4 mins margin of safety.
+    timestamp = (datetime.utcnow() - timedelta(minutes=4)).strftime('%Y-%m-%d %H:%M')
+    repo = entities.Repository(url=settings.repos.yum_1.url, product=module_prod).create()
```
[the remark this answers is not in the document]
Comment (swadeley) [reply tests/foreman/ui/test_repository.py:1067]: ACK

**Comment by swadeley** [comment]
trigger: robottelo
pytest: tests/foreman/ui/test_repository.py -k test_sync_status_persists

**Comment by mshriver** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_repository.py -k test_sync_status_persists

**Review by mshriver — APPROVED** [review]

**Comment by swadeley** [comment]
Thank you.

**Pull request merged**